apps/cookbook/src/app/showcase/list-load-on-demand-showcase/list-load-on-demand-showcase.component.ts

```diff
@@ -1,4 +1,6 @@
 import { Component } from '@angular/core';
+
+import { config } from '~/app/examples/list/components/load-on-demand/list-load-on-demand-example.component';
```

Comment:
Consider aligning with other showcase examples (get the template from the example instance in the markup)